### ISSUE TITLE
Help me in understanding of code.

### ISSUE THREAD

**Issue by mozillah** [lmoroney/dlaicourse#188]
I am working on Course 2 - Part 2 - Lesson 2 - Notebook but I could not understand why we are preprocessing the feature by using these operation in the notebook.if someone can explain me why we are doing it.I will be very thankful to you guys.

`for i in range(n_features):
        **x  = feature_map[0, :, :, i]
        x -= x.mean()
        x /= x.std ()
#         print(x)

        x *=  64
        x += 128
        x  = np.clip(x, 0, 255).astype('uint8')`**

![Screenshot from 2021-06-17 09-00-58](https://user-images.githubusercontent.com/71090794/122329527-03e88900-cf4b-11eb-83e3-23066f03aa31.png)

**Comment by smita-09** [comment]
Are you referring to Week2 of Course 2(Convolutional Neural Networks in TensorFlow)?

**Comment by mozillah** [comment]
> Are you referring to Week2 of Course 2(Convolutional Neural Networks in TensorFlow)?

Yes. I am referring to the course 2 of tensor flow specialization where we show filter output of heat map visualization
.
